package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@api3/chains",
+  "name": "@phase21/chains",
```

Comment:
I'm confused. Why this change to `main`?

Reply:
I will use `phase21` branch to release on `@phase21` organization while the main branch will remain as `@api3`. there may be some conflicts in the process.

Reply:
Yes. That's correct.